data/seed.go

```diff
@@ -0,0 +1,86 @@
+package data
+
+import (
+	"log"
+	"math/rand"
+	"context"
+	"io/ioutil"
+	"strings"
+
+	"github.com/pdeguing/empire-and-foundation/ent/planet"
+)
+
+func randomPlanetType(r *rand.Rand) planet.PlanetType {
+	n := r.Intn(9)
+
+	if n < 2 {
+		return planet.PlanetTypeHabitable
+	} else if n > 6 {
+		return planet.PlanetTypeGaseous
+	} else {
+		return planet.PlanetTypeMineral
+	}
+}
+
+func getPositionCode(r int, s int, o int, su int) int {
+	return su + o << 4 + s << 8 + r << 12
+
+}
+
+func randomPlanetSkin(r *rand.Rand) string {
+	files, err := ioutil.ReadDir("public/images/planet-dashboards")
```

Comment:
Smart. But maybe cache this somewhere so we don't read from the disk that many times. Not sure how much the OS caches this.